README.md

```diff
@@ -28,6 +28,24 @@ TypeScript-friendly interfaces are provided in `schemas/typescript/`:
 - `schemas/typescript/get-quote.ts`
 - `schemas/typescript/intent.ts`
 
+### Failure handling semantics
+
+Providers may include an optional `failureHandling` field on each quote to indicate how execution failures are managed. It supports:
```

Comment:
Why not make some sort of failure handling policy required? What can the caller assume if one is not included?